app/questionnaire/questionnaire_schema.py

```diff
@@ -336,12 +343,32 @@ def _update_answer_dependencies_for_value_source(
                     self._answer_dependencies_map[answer_id_for_block] |= {
                         self._get_answer_dependent_for_block_id(block_id=block_id, answer_id=answer_id)  # type: ignore
                     }
+        if value_source["source"] == "list":
+            section = self.get_section_for_block_id(block_id)
+            list_collector = self.get_list_collector_for_list(
+                section=section, for_list=value_source["identifier"]  # type: ignore
```

Comment:
as above

Reply:
This has a comment below it, I will move it above.

Reply:
ah my bad sorry!